client/src/lib/solana/sol-data.ts

```diff
@@ -84,28 +135,15 @@ export class SolData extends Assignable {
     return [W3ID_CONTEXT, SolData.solContext(version)];
   }
 
-  static sparse(
+  static async sparse(
```

Comment:
Don't see a reason for this to be async.

Reply:
That's true, It called empty before which is async so I added it then, and didn't remove it after